README.md

```diff
@@ -0,0 +1,185 @@
+
+
+```
+git clone https://github.com/rafapolmann/marajoara-cinema-management
+```
+
+#### Back-end editing
+You should just open **Marajoara.Cinema.Management.sln** file with **Visual Studio**.
+
+#### Front-end editing
+You should just open **Marajoara.Cinema.Management.FrontEnd** directory with **VS Code**, open a new Terminal Command tab and execute the command below:
+
+```
+npm install
+```
+
+## ⚙️ Debugging execution
+
+In this case the application will use SQL Server LocalDB.
+* You must execute **Marajoara.Cinema.Management.Api** with **Visual Studio**.
+* You need to change the **environment.ts** file in **Marajoara.Cinema.Management.FrontEnd** to use port: **44328** in baseApiUrl.
+* Open **Marajoara.Cinema.Management.FrontEnd** directory with **VS Code**, open a new Terminal Command tab and execute the command below:
+
+```
+ng serve
+```
+After above steps:
+* Access this address: http://localhost:4200/
+* Access the Swagger in this address: https://localhost:44328/swagger/index.html
+
+## 📦 Deployment
+
+### Docker Compose
+
+You must have Docker installed in your machine.
+
+* Open a **Terminal Command** in **Marajoara.Cinema.Management** directory (where is **docker-compose.yml** file).
+* You should create a folder in **c:/sqlvol/** or change the **docker-compose.yml** properties for a directory of your preference.
+* Execute in **Terminal Command** the following command:
+
+```
+docker-compose up -d
+```
+After above steps:
+* Access this address: http://localhost:4300/
+
+### Kubernetes (Kind)
+
+You must have Docker and KinD installed in your machine.
+
+* For creating a cluster, open a **Terminal Command** in **Marajoara.Cinema.Management\Kubernetes** (where are YAML configurations files)
+* You should create a folder in **c:/kubedata** or change the **create-cluster.yaml** properties for a directory of your preference.
+* Execute in **Terminal Command** the following commands:
+
+```
+kind create cluster --name {your-cluster-name} --config .\create-cluster.yaml
+```
+```
+kubectl apply -f .\PersistentVolume.yaml
+```
+```
+kubectl apply -f .\PersistentVolumeClaim.yaml
+```
+```
+kubectl apply -f .\SqlServerDeployment.yaml
+```
+```
+kubectl apply -f .\SqlServerService.yaml
+```
+```
+kubectl apply -f .\MarajoaraBackendDeployment.yaml
+```
+```
+kubectl apply -f .\MarajoaraBackendService.yaml
+```
+```
+kubectl apply -f .\MarajoaraFrontendDeployment.yaml
+```
+```
+kubectl apply -f .\MarajoaraFrontendService.yaml
+```
+
+After above steps:
+* Access this address: http://localhost:4400/
+
+#### Obs.: Cannot possibly make both deployment (Docker and Kind) simultaneously in the same machine because back-end API use the same port: 44500.
+
+### 🔩 First execution
+
+In the first time that you running the application, there wil not have registered user. You should click in ***Resgitre-se aqui*** to create an user account.
```

Comment:
On the first time running the application, there will be no user on the database. You should click in ***Registre-se aqui*** to create a user account.
OBS: The first user registered will have the Manager Access Level. The following will be registered as Customer.